generate-repo-cache.nu

```diff
@@ -50,11 +46,18 @@ def add-optional-attributes [
     |merge {$metadata}
 }
 
+do -i {rm repo-cache.nuon}
 
 ls modules
-|each {|module|
-    let metadata = (add-optional-attributes (get-metadata $module.name))
-    check-required-attributes $metadata
-    echo $metadata
+|where type == dir
+|get name
+|each {|it|
+    ls $it
+    |get name
+    |where $it =~ "*.nuon"
+    |each {|module|
+        check-required-attributes (add-optional-attributes (get-metadata $module))
+        |to nuon
+        |save repo-cache.nuon --append
```

Comment:
@Yethal any idea why does it get echo'ed instead of saving?

Reply:
I don't understand why the current solution fixed but it's fixed and I don't want to get into rabbit hole ig